app/views/insured/families/personal.js.erb

```diff
@@ -10,12 +10,12 @@
     demographicValidations();
 
   } else {
-    //manage family/ my household page
+    // manage family/ my household page
     var personContainer = $("#person-<%= @person.id %>")
     $("#person-<%= @person.id %> .append_consumer_info").html("<%= escape_javascript(render partial: "personal")%>");
     personContainer.removeClass('hidden');
     $("a[id^=edit-member]").addClass('disabled');
-    $("a[id^=edit-member]").atrr('disabled', true);
+    $("a[id^=edit-member]").attr('disabled', true);
```

Comment:
I don't think anchors even need the disabled attribute, the disabled class should suffice